modules/cert-manager-install-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * security/cert-manager-operator-install.adoc
+
+[id="cert-manager-install-console_{context}"]
+= Installing the {cert-manager-operator} using the web console
+
+You can use the web console to install the {cert-manager-operator}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Enter *{cert-manager-operator}* into the filter box.
+
+. Select the *{cert-manager-operator}* and click *Install*.
+
+. On the *Install Operator* page:
+.. Select the *tech-preview* channel as the *Update channel*. The *tech-preview* channel installs the latest Technology Preview release of the {cert-manager-operator}.
+.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the Operator-recommended `openshift-cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
+.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `openshift-cert-manager-operator`.
++
+If the `openshift-cert-manager-operator` namespace does not exist, it is created for you.
+.. Click the *Enable Operator recommended cluster monitoring on the Namespace* checkbox to enable cluster monitoring for the Operator.
+.. Select an *Update approval* strategy.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.Verification
+
+. Navigate to *Operators* -> *Installed Operators*.
+. Verify that *{cert-manager-operator}* is listed with a *Status* of *Succeeded*.
+
+////
+Questions:
+ - When I click the operator after it's been installed, there are two "CertManager" tabs. One has an instance of "cluster", the other is empty.
+ - Need to create an instance like the descheduler does? The one CertManager tab already has one ^
```

Comment:
@slaskawi These questions here I'm still unsure about. The two duplicate tabs is confusing, so let me know if anything is going to change on that front or not.

I watched your demo in the TE slides, and I can see you manually creating the CertManager CR, but as I say here, after I install the operator and come here, there is already a "cluster" instance installed. So I'm not sure if this step is necessary, or what we need to say in the docs.

Reply:
Yes and no.

You can create a CR instance yourself if you wish. That will work just fine. However, the operator will create a default CR if you don't supply any. Unfortunately, we had to implement it this way because the `library-go` assumes that the CR is always there. 

In the future this may potentially change. But for now - it is the way it is.

Reply:
@slaskawi so since the operator is creating a default CR for you, I'm not going to add any steps to tell users to create one - let me know if this is okay or not

Reply:
Got answer in slack - won't tell users to create one